packages/expo-video/src/VideoView.web.tsx

```diff
@@ -24,6 +24,13 @@ function mapStyles(style: VideoViewProps['style']): React.CSSProperties {
   return flattenedStyles as React.CSSProperties;
 }
 
+export function isPictureInPictureSupported(): boolean {
+  const userAgent = window.navigator.userAgent;
+  // Chromium and WebKit based browsers are supported
+  // https://developer.mozilla.org/en-US/docs/Web/API/Picture-in-Picture_API#browser_compatibility
+  return !!userAgent && (userAgent.includes('Chrome') || userAgent.includes('Safari'));
```

Comment:
For sure there is another smarter way to check whether PiP is supported. Depending on user agent is almost never a good option. Maybe something like:
```suggestion
  return typeof document === 'object' && typeof document.exitPictureInPicture === 'function';
```